eng/pipelines/macos.yml

```diff
@@ -0,0 +1,28 @@
+# Linux legs
+jobs:
+
+# Legs with Helix testing
+- template: corefx-base.yml
+  parameters:
+    name: MacOS
+    displayName: MacOS
+    strategy:
+      matrix:
+        x64_Debug:
+          _configuration: Debug
+          _architecture: x64
+          framework: netcoreapp
+          helixQueues: $(macOSQueues)
+
+    pool:
+      name: Hosted MacOS
+
+    targetOS: OSX
+
+    preBuildSteps:
+      - script: |
+          brew install pkgconfig openssl
```

Comment:
Is there a plan to get these baked-in to the mac machines so we don't have to install them every time?